File: app/src/main/java/org/dhis2afgamis/usescases/about/AboutComponent.java
package org.dhis2afgamis.usescases.about;

import org.dhis2afgamis.data.dagger.PerFragment;

import dagger.Subcomponent;

/**
 * QUADRAM. Created by ppajuelo on 05/07/2018.
 */
@PerFragment
@Subcomponent(modules = AboutModule.class)
public interface AboutComponent {
    void inject(AboutFragment programFragment);

}

File: app/src/main/java/org/dhis2_haparent/usescases/about/AboutComponent.kt
package org.dhis2_haparent.usescases.about

import dagger.Subcomponent
import org.dhis2_haparent.commons.di.dagger.PerFragment

@PerFragment
@Subcomponent(modules = [AboutModule::class])
interface AboutComponent {
    fun inject(programFragment: AboutFragment?)
}
